File: auth-service/src/main/java/com/eirsteir/coffeewithme/authservice/AuthServiceApplication.java
package com.eirsteir.coffeewithme.authservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@EnableEurekaClient
@EnableJpaRepositories(basePackages = "com.eirsteir.coffeewithme.authservice.repository")
@ComponentScan(
    basePackages = {
      "com.eirsteir.coffeewithme.commons.exception",
      "com.eirsteir.coffeewithme.authservice"
    })
@SpringBootApplication
public class AuthServiceApplication {

  public static void main(String[] args) {
    SpringApplication.run(AuthServiceApplication.class, args);
  }
}

File: auth-service/src/main/kotlin/com/eirsteir/coffeewithme/authservice/AuthServiceApplication.kt
package com.eirsteir.coffeewithme.authservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.context.annotation.ComponentScan

@EnableEurekaClient
@ComponentScan(basePackages = ["com.eirsteir.coffeewithme.commons.exception", "com.eirsteir.coffeewithme.authservice"])
@SpringBootApplication
class AuthServiceApplication

fun main(args: Array<String>) {
    runApplication<AuthServiceApplication>(*args)
}